src/pages/pdf-pricing.md

```diff
@@ -36,13 +36,13 @@ Collaboration and analytics
 
 <TitleBlock slots="heading" theme="light" />
 
-## Adobe PDF Services API
+## <div id = "adobe-pdf-services-api">Adobe PDF Services API</div>
```

Comment:
Just a minor comment: please use `span` instead so the final HTML output will be : 
```
<h2> <span id="anchor">text</span> </h2>
```
It's better to have an inline element inside a heading element, from a semantic point of view.